### PR TITLE
Create Shopify podcast post

### PR DESCRIPTION
**Pull request by rebeccadee**
https://app.asana.com/0/1200356056235926/1200770286063273/f

### CODE REVIEW

File: podcast/3.1.md [hunk 1]
```diff
@@ -0,0 +1,429 @@
+---
+title: "Decomposing a massive Rails monolith with Kirsten Westeinde, software development manager, Shopify"
```
Comment (rebeccadee) [inline podcast/3.1.md:2]: @content-hacker I've suggested this title, feel free to reword!

**Comment by rebeccadee** [comment]
@content-hacker One question for you here and re: the teasers in the doc: https://docs.google.com/document/d/1SNcPVzC2olRB8MWHWxP0ROT-I9q0AseEOY_XwADsRhk/edit?disco=AAAAN-pBgmE

Thanks!

**Comment by content-hacker** [comment]
@rebeccadee FYI I removed the episode intro from the transcript so that the transcript begins with the start of the episode (we'll want to follow this format for future episodes as well).

File: podcast/3.1.md [hunk 2]
```diff
@@ -0,0 +1,426 @@
...
+<!-- END AUDIO -->
+
+<!-- START GUESTS -->
+<span>Kirsten Westeinde, Beyang Liu</span>
+<!-- END GUESTS -->
+
+<!-- START SUMMARY -->
+What's it like to deconstruct one of the largest Rails codebases (3 million lines of code, 500,000+ lifetime commits, 40,000 files) on the planet? And why didn't Shopify follow the standard path to microservices, but instead choose to modularize their monolith? 
+
+In this episode, Kirsten Westeinde, software development manager at Shopify, describes how her team led the charge in refactoring and re-architecting Shopify's massive codebase, sharing the winding path they took to make this massive change and the way they tackled both the technical and human sides of this challenge.
```
Comment (rebeccadee) [inline podcast/3.1.md:20]:
```suggestion
In this episode, [Kirsten Westeinde](https://www.kirstenwesteinde.com/), software development manager at Shopify, describes how her team led the charge in refactoring and re-architecting Shopify's massive codebase, sharing the winding path they took to make this massive change and the way they tackled both the technical and human sides of this challenge.
```

**Pull request merged**